steps/standard_ruleset_step_definitions.py

```diff
@@ -1,122 +1,122 @@
 from datetime import datetime
 import re
 
+from behave import given, then
 from bdd_tester.exceptions import StepException
 
 
 @given('an IATI activity')
-def step_impl(context):
+def step_given_iati_activity(context):
     # this is a dummy step. It's here because
```

Comment:
@andylolz I've taken the liberty of changing and given different names to the step functions. I think it is the way Behave goes about it, but I find it quite odd (in Python at least). My editor with flake8 keeps complaining about it and probably any CI setup with flake8 in it will complaint too!

Reply:
Just to clarify: I'm referring as 'odd' to the fact of having functions with the same name in one script.

Reply:
Haha, yeah fair enough. I don’t have strong feelings on this either way, so happy to break the convention and make function names a bit more explanatory.

More generally: my idea was really that this repo could exist independent of the features that it tests – like behave does. So any features & steps contained within this repo would really be for demonstration purposes only. So how steps are defined, then, is really at the discretion of the user :)